src/bramble/client/ProjectLimiter.js

```diff
@@ -0,0 +1,69 @@
+define(function() {
+    "use strict";
+    var ProjectLimiter = {};
+
+    var _maxProjectState = {
+            fileSize: 0, // max size of any file in bytes. NOT USED now.
+            fileCount: 0, // total number of files you can have in a project
+            projectSize: 0// total number of bytes for all files in this project
+    };
+
+    var _currentProjectState = {
+        numOfFiles: 0 ,
+        projectSize: 0
+    };
+
+    var _fs;
+
+    ProjectLimiter.initMaxLimits =  function (projectState,fs){
```

Comment:
NOTE: we'll want to have `init` take a `callback` function, since this is going to need to happen asynchronously.